agent/structs/check_definition.go

```diff
@@ -42,6 +43,96 @@ type CheckDefinition struct {
 	OutputMaxSize                  int
 }
 
+func (t *CheckDefinition) UnmarshalJSON(data []byte) (err error) {
+	type Alias CheckDefinition
+	aux := &struct {
+		// Parse special values
+		Interval                       interface{}
+		Timeout                        interface{}
+		TTL                            interface{}
+		DeregisterCriticalServiceAfter interface{}
+
+		// Translate fields
+		ArgsCamel                           []string    `json:"args"`
+		ScriptArgsCamel                     []string    `json:"script_args"`
```

Comment:
Should field names in the unmarshallers responsible for parsing the snake case name variant be suffixed with `Camel`?

Reply:
done